cgi-bin/ljtestlib.pl

```diff
@@ -0,0 +1,83 @@
+package LJ;
+
+use strict;
+no warnings 'uninitialized';
+
+
+BEGIN {
+    use Test::MockObject;
+
+    my $mock = Test::MockObject->new();
+
+    sub fake_lang_ml {
+	my ( $code, $vars ) = @_;
```

Comment:
Looks like this file never got tidied, this is the second failure in the build! You can do this from your local installation by doing something like:

`perl -I$LJHOME/extlib/lib/perl5 $LJHOME/extlib/bin/tidyall -a`

(Tidy is basically a Perl formatter we use, to ensure that code stays readable!)

Reply:
done